web/static/js/components/questionnaires/QuestionnaireTitle.jsx

```diff
@@ -34,14 +34,20 @@ class QuestionnaireTitle extends Component {
     return (
       <div className='title-container'>
         <EditableTitleLabel title={questionnaire.name} onSubmit={(value) => { this.handleSubmit(value) }} emptyText={t('Untitled questionnaire')} readOnly={readOnly} more={<QuestionnaireMenu />} />
-        <EditableDescriptionLabel description={questionnaire.description} emptyText={t('Add description')} onSubmit={(value) => { this.handleSubmitDescription(value) }} readOnly={readOnly} />
+        <EditableDescriptionLabel description={questionnaire.description} emptyText={readOnly ? '' : t('Add description')} onSubmit={(value) => { this.handleSubmitDescription(value) }} readOnly={readOnly} />
       </div>
     )
   }
 }
 
 const mapStateToProps = (state, ownProps) => ({
-  readOnly: state.project && state.project.data ? state.project.data.readOnly : true
+  readOnly: typeof ownProps.readOnly === 'boolean'
```

Comment:
why this change?

Reply:
Because we don't want the title to be changed from the simulation page.

It's related to [this other change](https://github.com/instedd/surveda/pull/1698/commits/621d2946311c1e0361cf60df7cd00916bd456ddb#diff-628a34867fbc251edf9230cc7c96e629R66).

So, the intention here was: if the component's user explicitly decides to set it read-only (or the opposite) prioritize that.

Reply:
ok